docs/packages/Parser.mdx

```diff
@@ -423,11 +423,9 @@ This option allows you to toggle the warning that the parser will give you if yo
 
 WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.
 
-You may find that it works just fine, or you may not.
-
-SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <5.1.0
-
-YOUR TYPESCRIPT VERSION: 5.1.3
+* @typescript-eslint/typescript-estree version: 8.14.0
+* Supported TypeScript versions:: 5.7.1
+* Your TypeScript version: >=4.7.4 <5.7.0
```

Comment:
These are the wrong way around aren't they?

Reply:
Yes, I swapped typescript version range and the active version 😩. Should be fixed now!

Reply:
😩 I totally missed this! 😂